core/src/main/scala/com/nvidia/spark/rapids/tool/qualification/QualificationArgs.scala

```diff
@@ -112,6 +112,11 @@ Usage: java -cp rapids-4-spark-tools_2.12-<version>.jar:$SPARK_HOME/jars/*
     opt[Boolean](required = false,
       descr = "Whether to parse ML functions in the eventlogs. Default is false.",
       default = Some(false))
+  val ignoreTransitions: ScallopOption[Boolean] =
+    opt[Boolean](required = false,
+      descr = "Whether to ignore durations for ColumnarToRow and RowToColumnar transitions " +
```

Comment:
maybe rename to penalizeTransitions and have it true by default.

Whether to add a penalty for ColumnarToRow and RowToColumnar transitions. Default is true.